text/0015-expose-handeled-property.md

```diff
@@ -0,0 +1,43 @@
+* Start Date: 2022-09-26
+* RFC Type: feature
+* RFC PR: https://github.com/getsentry/rfcs/pull/15
+
+# Summary
+
+Allow a user to decide if an exception/error is considered `handled: true` or `handled: false`.
```

Comment:
I do not get the "why" for this request. If an exception is caught by the global exception handler of the SDK it is considered not handled. If the user catches the exception himself/herself and then sends it to Sentry with `captureException()` it is considered as handled, because it was handled by the user.

I think this is how it should work.

Reply:
A PHP application can only register one `error_handler` at a time.
The `error_handler` we ship with the Sentry SDK is therefore often overwritten by the framework `error_handler`, hence we have to hook into it. And then we can't access the `ExceptionMechanism` anymore.